solution/ui/regulations/eregs-vite/src/views/Search.vue

```diff
@@ -355,24 +355,22 @@ export default {
         },
         async retrieveResourcesResults({ query, page, pageSize }) {
             try {
-                const isSearchGov = window.location.href.match("searchgov");
-
                 const commonParams = {
                     q: query,
                     page,
                 };
 
-                const djangoParams = {
-                    ...commonParams,
-                    partDict: "all",
-                    page_size: pageSize,
-                    fr_grouping: false,
-                };
-
-                const response = isSearchGov
-                    ? await getSearchGovResources(commonParams)
-                    : await getSupplementalContent(djangoParams);
+                let response = await getSearchGovResources(commonParams)
 
+                if(response.count === 0){
+                    const djangoParams = {
+                        ...commonParams,
+                        partDict: "all",
+                        page_size: pageSize,
+                        fr_grouping: false,
+                    };
+                    response = await getSupplementalContent(djangoParams);
```

Comment:
If we are changing this behavior, it would be nice to add some test coverage to make sure it works as expected.  We could specifically intercept these two requests and return some new fixtures that have 0 results for the `getSearchGovResources` interception and has > 0 results for the `getSupplementalContent` interception.